app/assets/scripts/components/map.js

```diff
@@ -264,11 +264,22 @@ const Map = React.createClass({
     const t = get(window.t, [this.props.lang, 'map_labels'], {});
     return (
       <span className='legend__markers'>
-        <span className='legend__item legend__marker--cluster'><span className='legend__image legend__image--cluster'>
-          <span className='legend__image--cluster--bg'></span>
-          <span className='legend__image--cluster--text'>8</span>
-        </span>{t.map_group_label}</span>
-        <span className='legend__item legend__marker--project'><span className='legend__image legend__image--marker'><img src='assets/graphics/content/map-pin.png' alt='A marker indicates a single project'/></span> {t.map_project_label}</span>
+        <span className='legend__item legend__marker--cluster'>
+          <span className='legend__image legend__image--cluster'>
+          <span className='legend__image--cluster--bg'>
+          </span>
+          <span className='legend__image--cluster--text'>
+            8
```

Comment:
Is there a reason why we're hard coding this to `8` instead of counting the number of markers on the page?